client/src/app/site/motions/components/motion-detail/motion-detail.component.ts

```diff
@@ -137,24 +139,42 @@ export class MotionDetailComponent extends BaseComponent implements OnInit {
         });
     }
 
+    /**
+     * determine the motion to display using the URL
+     */
+    public getMotionByUrl(): void {
+        if (this.route.snapshot.url[0] && this.route.snapshot.url[0].path === 'new') {
+            // creates a new motion
+            this.newMotion = true;
+            this.editMotion = true;
+            this.motion = new ViewMotion();
+            this.motionCopy = new ViewMotion();
+        } else {
+            // load existing motion
+            this.route.params.subscribe(params => {
+                this.repo.getViewModelObservable(params.id).subscribe(newViewMotion => {
+                    this.motion = newViewMotion;
+                });
+            });
+        }
+    }
+
     /**
      * Async load the values of the motion in the Form.
      */
     public patchForm(formMotion: ViewMotion): void {
-        this.metaInfoForm.patchValue({
-            category_id: formMotion.categoryId,
-            supporters_id: formMotion.supporterIds,
-            submitters_id: formMotion.submitterIds,
-            state_id: formMotion.stateId,
-            recommendation_id: formMotion.recommendationId,
-            identifier: formMotion.identifier,
-            origin: formMotion.origin
+        const metaInfoControls = Object.keys(this.metaInfoForm.controls);
+        const metaInfoPatch = {};
+        metaInfoControls.forEach(ctrl => {
```

Comment:
`Object.keys(this.metaInfoForm.controls).forEach(ctrl => {` and remove the `metaInfoControls` variable.